bootique/src/main/java/io/bootique/help/config/HelpConfigCommand.java

```diff
@@ -32,13 +32,17 @@
 
 public class HelpConfigCommand extends CommandWithMetadata {
 
+    public static final String HELP_CONFIG_OPTION = "help-config";
+
     private BootLogger bootLogger;
     private Provider<ConfigHelpGenerator> helpGeneratorProvider;
 
     public HelpConfigCommand(BootLogger bootLogger, Provider<ConfigHelpGenerator> helpGeneratorProvider) {
         super(CommandMetadata
                 .builder(HelpConfigCommand.class)
-                .description("Prints information about application modules and their configuration options.")
+                .description("Prints information about application modules and their configuration options. "
+                        + "Optionally, you can provide a configName to print only that config.")
+                .valueOptional("configName")
```

Comment:
Since we aren't changing the filtering yet, I updated this to reflect current functionality.